### PR TITLE
feat(api): add new speed for emulsifying pipette plunger

### PR DESCRIPTION
**Pull request by caila-marashaj**
## Overview
Add a higher speed configuration for the peek pipettes, which will want to move the pipette axis at a max speed of 90, rather than the normal 70 mm/sec.

## Changelog

- add a new `highSpeed` quirk, and handling for it in the `Pipette` class
- change the `uiMaxFlowRate` in shared data to reflect the new max speed
- when we attach a pipette, check if it has a `highSpeed` quirk
- if so, update the motion constraints to reset only the plunger axis' max speed to 90 mm/s
- specify the new speed in `defaults_ot3`
- add backend functions to change the motion constraints
- couple tests

### CODE REVIEW

**Comment by ryanthecoder** [comment]
Can we add a test to make sure it works in motion planning? and can we update the shared data def to increase the "uiMaxFlowRate", you may need to do some math to figure out what that is now or ask one of the hardware people

**Review by sfoster1 — CHANGES_REQUESTED** [review]
We should not do logic on display names; those are for human consumption only, and we want the ability to change them for branding reasons or whatever without constraint. Instead, we should use the pipette name (this is "p1000 8 channel" as opposed to "p1000 8 channel v3.3") or something like that. We can add more data or logic if we need to support this, but display names should never show up in logic.

File: api/src/opentrons/hardware_control/ot3api.py [hunk 1]
```diff
@@ -634,10 +634,21 @@ async def cache_pipette(
             self._feature_flags.use_old_aspiration_functions,
         )
         self._pipette_handler.hardware_instruments[mount] = p
+        if not self.is_simulator:
```
Comment (sfoster1) [inline api/src/opentrons/hardware_control/ot3api.py]: why do we need `self.is_simulator`?

**Review by sfoster1 — APPROVED** [review]
If this needs to get merged right now, let's merge it and then immediately start work on removing that `if not self.is_simulator` thing - there's no reason we should have to have that.

**Pull request merged**